src/app/api/v1/fxa-rp-events/route.ts

```diff
@@ -57,7 +64,9 @@ const getJwtPubKey = async () => {
  * @param {NextRequest} req
  * @returns {Promise<jwt.JwtPayload>} decoded JWT data, which should contain FxA events
  */
-const authenticateFxaJWT = async (req: NextRequest) => {
+const authenticateFxaJWT = async (
```

Comment:
nit: (although prior art) Should this be "JWT" or "Jwt" to be consistent w/ L39 above (and the `JwtPayload` references)?

Reply:
fixed thanks